modules/manage/pages/manage-xdcr/xdcr-differ.adoc

```diff
@@ -0,0 +1,58 @@
+= xdcrDiffer Utility
+:description: pass:q[The `xdcrDiffer` utility helps you verify data consistency between XDCR clusters by comparing document metadata and values, and reporting missing or mismatched documents.]
+
+[abstract]
+{description}
+
+`xdcrDiffer` is a diagnostic utility included in the Server installation package for convenient access.
+Previously, you'd build this utility from source using the xdcrDiffer GitHub repository.
```

Comment:
Could we link the https://github.com/couchbase/xdcrDiffer to the text "xdcrDiffer Github repository"?

Previously, you'd build this utility from source using the [xdcrDiffer GitHub repository](https://github.com/couchbase/xdcrDiffer).